sql/core/src/main/scala/org/apache/spark/sql/api/python/PythonSQLUtils.scala

```diff
@@ -39,6 +40,11 @@ private[sql] object PythonSQLUtils {
     FunctionRegistry.functionSet.flatMap(f => FunctionRegistry.builtin.lookupFunction(f)).toArray
   }
 
+  def listSQLConfigs(): Array[(String, String, String)] = {
+    val conf = new SQLConf()
+    conf.getAllDefinedConfs.toArray
```

Comment:
The original `Seq` wouldn't translate over into Python, so I had to convert this to an `Array`. I wonder if there's a cleaner way to do this.

Reply:
This is fine but I would leave a comment though.